code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -18,6 +18,8 @@
 /mob/living/carbon/human/bee_friendly()
 	if(dna?.species?.id == "pod") //bees pollinate plants, duh.
 		return 1
+	if(dna?.species?.id == "apid") //bees don't attack bees, why would they?
```

Comment:
pretty sure there is a helper for that
`isapid()`

Reply:
I am having problems with the isapid() helper, do you think I can stick with the species id check?

Reply:
Way easier would be to give apids TRAIT_BEEFRIEND as an inherent trait

Reply:
I suggest changing line 19 to:
if(isapid(src) || ispodperson(src))

Reply:
try something like this
![image](https://user-images.githubusercontent.com/7960287/116946824-73483900-ac7b-11eb-8a9c-73b4acf2abb2.png)

Reply:
Another solution would be to just use the already existing TRAIT_BEEFRIEND and just give that to both apids and podpeople.